doc/components.md

```diff
@@ -138,6 +138,9 @@ your widgets, when you don't want the component to respect the camera nor the vi
 for example be for controls or joysticks that would be unergonomic to use if they had to stay within
 the viewport.
 
+Do note that this setting is only respected if the component is added directly to the root
+`FlameGame` and not as a child component of another component.
```

Comment:
So, basically, we have an option to respect the camera, but we don't always respect that option. Thus, there could be a situation where we respect the camera, but do not respect the respect of a camera, or vice versa, where we do not respect the camera but we do respect the disrespect of the camera. Should there be an option then to respect the respect of the respect of a camera?

Reply:
I'm guessing this is a rhetorical question? 😛
We could build it differently, but it would be pretty weird to have a child component of something that respects the camera to not respect the camera anyways, I can't see a sensible component structure like that at least.

Reply:
(i was joking)

Reply:
Haha, very hard to get whether someone is joking or not in text sometimes. :sweat_smile: